core/src/main/java/hudson/cli/handlers/ViewOptionHandler.java

```diff
@@ -73,10 +75,25 @@ public int parseArguments(Parameters params) throws CmdLineException {
         return 1;
     }
 
-    private View getView(String name) throws CmdLineException {
+    /**
+     *
+     * Gets a view by its name
+     *
+     * @param name A view name
+     * @return The {@link View} instance. Null if {@link Jenkins#getInstance()} returns null.
+     * @throws CmdLineException
+     *      If view isn't found or an un-expected error occurred
+     * @since TODO
+     */
+    @CheckForNull
+    public View getView(final String name) throws CmdLineException {
```

Comment:
> The method will throw RuntimeException if a user has no read permissions to the view. I suppose hasPermission() and null result should work

I do not agree. The method signals all the other possible failures via exception. I see no reason to use `null` return value for one of them. I suggest to document `AccessDeniedException` that will e thrown in case the access is denied.